src/components/canvas/Canvas.js

```diff
@@ -107,6 +121,8 @@ class Canvas extends Component {
   };
 
   render() {
+    const { house } = this.state;
```

Comment:
here you can extract multiple properties in the same call:

```
const { house, tree } = this.state;
```

Reply:
Great, thanks! I'll do it.